commit 7885aa76c86ab2cf7a06600506680d1976f8971f
Author: Scott Talbert <swt@techie.net>
Date:   Mon Feb 3 19:06:07 2014 -0500

    Include PTHREAD_CFLAGS when linking libhidapi-libusb
    
    Seems to be required as documented in ax_pthread.m4.  Resolves issue with
    Debian package build not finding the pthread symbols.

diff --git a/libusb/Makefile.am b/libusb/Makefile.am
index 7f6c9dc..d0c17eb 100644
--- a/libusb/Makefile.am
+++ b/libusb/Makefile.am
@@ -3,7 +3,7 @@ AM_CPPFLAGS = -I$(top_srcdir)/hidapi $(CFLAGS_LIBUSB)
 if OS_LINUX
 lib_LTLIBRARIES = libhidapi-libusb.la
 libhidapi_libusb_la_SOURCES = hid.c
-libhidapi_libusb_la_LDFLAGS = $(LTLDFLAGS)
+libhidapi_libusb_la_LDFLAGS = $(LTLDFLAGS) $(PTHREAD_CFLAGS)
 libhidapi_libusb_la_LIBADD = $(LIBS_LIBUSB)
 endif
 
